go/stream/message.go

```diff
@@ -0,0 +1,74 @@
+package stream
+
+import (
+	"encoding/json"
+
+	"github.com/go-playground/validator/v10"
```

Comment:
[non-blocking] I never used a struct validator before. Is this the most popular one? Are there more available possibly with benchmarks? When I googled it I came across this (closed) [issue](https://github.com/go-validator/validator/issues/78) so I now wonder :sweat_smile: